components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -106,15 +223,196 @@ class WidgetConfigurationPanel extends React.Component {
+                                }}
+                                value={this.state.options[i].defaultData}
+                                id={options[i].id}
+                                className="options-list">
+                                {items}
+                            </SelectField>
+                        </div>);
+                        break;
+                    case "TypeBoolean":
+                        preferences.push(<div className="options-list">
+                            <Checkbox id={options[i].id}
+                                      label={options[i].title}
+                                      labelStyle={labelStyle}
+                                      onCheck={(event, isInputChecked) => {
+                                          this.handlePreferenceCheckBoxEvent(event, isInputChecked, options), this.props.setWidgetConfigPanelDirty(true);
+                                      }}
+                                      checked={this.state.options[i].defaultData}
+                                      className="options-list"/>
+                        </div>);
+                        break;
+                    default :
+                        break;
+                }
+            }
+        }
+
+        return preferences;
+    }
+
+    getWidgetConfPanelContent() {
+        return (<div>
+            <List style={listStyle}>
+                <Subheader style={stylePanelHeader} className="options-list-header">Widget Configuration</Subheader>
+                <ListItem
+                    primaryText="Publishers"
+                    initiallyOpen={true}
+                    primaryTogglesNestedList={true}
+                    nestedItems={this.getPublishers()}
+                    nestedListStyle={itemBackgroundStyle}
+                    className="options-list-header"
+                    style={styleListHeader}
+                />
+                <Divider style={dividerStyle} inset={false}/>
+                <ListItem
+                    primaryText="Options"
+                    initiallyOpen={true}
+                    primaryTogglesNestedList={true}
+                    nestedItems={this.getPreferences()}
+                    nestedListStyle={itemBackgroundStyle}
+                    className="options-list-header"
+                    style={styleListHeader}
+                />
+            </List>
+        </div>);
+    }
+
+    WidgetConfDirtyNotifier() {
+        const actions = [
+            <FlatButton
+                label="No"
+                primary={true}
+                onClick={() => {
+                    this.props.handleCloseWithNo()
+                }}
+            />,
+            <FlatButton
+                label="Yes"
+                primary={true}
+                onClick={() => {
+                    this.props.handleCloseWithYes()
+                }}
+            />,
+        ];
+
+        return (
+            <div>
```

Comment:
remove the divs if not necessary

Reply:
fixed with 87b573730d4c8021f11a7fc89783b583eed6edbf